webservices/args.py

```diff
@@ -223,6 +223,8 @@ def make_seek_args(field=fields.Int, description=None):
 efilings = {
     'file_number': fields.List(fields.Int, description=docs.FILE_NUMBER),
     'committee_id': fields.List(IStr, description=docs.COMMITTEE_ID),
+    'report_id': fields.Int(),
+    'receipt_date': fields.Date(),
```

Comment:
we already have `min_reciept_date` and `max_reciept_date` so I don't think we want `receipt_date` too